src/py/flwr/common/exit/exit.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2025 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Unified exit function."""
+
+
+from __future__ import annotations
+
+import sys
+from logging import ERROR, INFO
+from typing import NoReturn
+
+from flwr.common import EventType, event
+
+from ..logger import log
+from ..version import package_version
+from .exit_code import EXIT_CODE_HELP, ExitCode
+
+HELP_PAGE_URL = f"https://flower.ai/docs/framework/{package_version}/en/ref-exit-codes/"
```

Comment:
This might output broken URLs for pre-release versions. For example, package version could be `1.15.0rc2`, but we don't publish `flower.ai/docs/framework/1.15.0rc`. At the point when this pre-release version is released, only `/1.14.0/` and `/main/` are available.